### DISCUSSION TITLE
Is it possible to show `total` for the list of users?

### DISCUSSION THREAD

**Discussion by aleksei-commmune** [AlexSim93/pull-request-analytics-action#37] [Q&A] [answered]
Hi! I see there are options `SHOW_USERS` and `HIDE_USERS` and I assume it's possible to make `SHOW_USERS`: total for total statistics only and use `HIDE_USERS` to exclude some users from it (please correct me if I'm wrong).
Is it possible to use `SHOW_USERS`: total + list the users to include in this total instead of excluding?

**Comment by AlexSim93** [comment, marked as answer]
Hi! Yeah, you can use these parameters to hide users from the report. And yes, you can use user's nicknames + total - it will show stats of mentioned users + total.

But remember, it just hides results, total stats will include even hidden user results

**Reply by aleksei-commmune** [reply]
Hi, thank you for the answer!
Is there a way to exclude users from the total? 
E.g. we have a monorepo for multiple teams, but I want to measure only our team's metrics. Also, I don't want to measure the metrics of each person, only the team's total. 
Is this possible?

**Reply by AlexSim93** [reply]
At this moment, there is only one way to do it. By adding labels to PRs and using INCLUDE_LABELS or EXCLUDE_LABELS.

Showing GitHub teams stats is one of the goals. Probably it will be done during summer.

**Reply by aleksei-commmune** [reply]
I got it, thanks!

**Reply by AlexSim93** [reply]
Hi, team support has been released! Use `v3` version to check it out.
